deploy/role.yaml

```diff
@@ -31,6 +31,7 @@ rules:
   - get
   - create
   - update
+  - patch
```

Comment:
hopefully we already have `oc apply -f ./deploy/role.yaml` ))
![Screenshot_20200423_113726](https://user-images.githubusercontent.com/5887312/80078390-86e49e00-8557-11ea-8579-097b74479c98.png)

Reply:
Finally figured this out: we had conflicting definitions of the `che-workspace-controller` clusterrole. `kubectl apply`ing the entire folder would create the role we meant, and then overwrite it. 

I've reworked a the role/rolebinding definitions to a single yaml file now.